recipes/usalign/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "USalign" %}
+{% set version = "20240303" %}
+{% set sha256 = "b4da9fc58449f96a98eabf4137082cfc161787a5df739df351186e70b9c87950" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://zhanggroup.org/US-align/bin/module/USalignLinux64.zip
```

Comment:
https://zhanggroup.org/US-align/bin/module/USalign.cpp

It should be this address, so we don't download the precompiled binary?